src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -221,7 +227,10 @@ protected override string GenerateCommandLineCommands ()
 
 		protected override string GenerateFullPathToTool ()
 		{
-			return Path.Combine (ToolPath, ToolExe);
+			var fullToolPath = Path.Combine (ToolPath, ToolExe);;
+			if (File.Exists (fullToolPath))
+				return fullToolPath;
+			return Path.Combine (ToolPath, "bin", ToolExe);
```

Comment:
If the ToolPath is set to empty (by the user) this code path will get hit. So we kinda need the same code. I will move it into one method